lib/src/utils/helpers.js

```diff
@@ -363,6 +363,17 @@ export const populateWithDummyValues = (obj, presets) => {
 }
 
 export const renderPageData = async (app, pageName) => {
+
+  try {
+    let hooksJavascript = importJs(`/js/global-hooks`)
+    if (hooksJavascript['init_params']) {
+      const paramsUpdate = hooksJavascript['init_params'](app.context.$sections, { qs: app.context.route.query, headers: app.context && app.context.req ? app.context.req.headers : {}, reqBody: app.context && app.context.req ? app.context.req.body : {} })
```

Comment:
same here, can't see the request url.

Also that's not DRY.. can't you centralize these hooks in one file and call them from the different places in your code where they need to be invoked ?